packages/web-pkg/src/components/AppTopBar.vue

```diff
@@ -158,38 +156,13 @@ export default defineComponent({
       return $gettext(`Autosave (every %{ duration })`, { duration: duration.toHuman() })
     })
 
-    const { getParentFolderName, getParentFolderLinkIconAdditionalAttributes, getPathPrefix } =
-      useFolderLink()
-
-    const space = computed(() => getMatchingSpace(props.resource))
-
-    //FIXME: We currently have problems to display the parent folder name of a shared file, so we disabled it for now
-    const isPathDisplayed = computed(() => {
-      return !isShareSpaceResource(unref(space)) && !isPublicSpaceResource(unref(space))
```

Comment:
The public space condition is still needed, otherwise the Location shows "Personal" on public pages.